### ISSUE TITLE
NET Standard

### ISSUE THREAD

**Issue by nosmirck** [ufuf/AdvancedTimer#20]
Hello! Is there any way to make this compatible with NET Standard (1.6 and up)?

I can try doing it myself, but if can someone point me in the right direction, I'll be thankful!

Thanks in advance!

**Comment by ufuf** [comment]
Hi @nosmirck 
I am not maintaining this repo anymore.
I would be happy if you can fork and do it.
But I am not familiar with porting this to .Net Standart actually. 
Best,
Ufuk

**Comment by nosmirck** [comment]
Great, I'll fork and maybe Pull Request after I upgrade? I don't know how this Nuget thing works, so if you accept my PR maybe the library can be alive again in the Nuget repo.

I'll report back here again as soon as I have it working with NET Standard.

**Comment by ufuf** [comment]
Just send a PR and I will accept it.
You can check this link for creating nuget.
https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package

**Comment by MichaelLant** [comment]
@nosmirck Any progress on porting to .Net Standard?

**Comment by d-m-wilson** [comment]
I forked this project and migrated it to .NET Standard 2.0. Be aware, however, that I removed the iOS legacy and Windows Phone projects, as I am only able to work with Android and iOS Unified. https://github.com/d-m-wilson/AdvancedTimer https://www.nuget.org/packages/info.dmwilson.AdvancedTimer.Forms.Plugin At this point I am considering my builds to be _alpha_ quality and not ready for production. I am about to use the latest build in an app, and if all goes well then I will upgrade it to _beta_ status.
